static/app/components/onboarding/productSelection.spec.tsx

```diff
@@ -228,37 +223,7 @@ describe('Onboarding Product Selection', function () {
     expect(screen.getByRole('checkbox', {name: 'Session Replay'})).toBeDisabled();
   });
 
-  // TODO: This test does not play well with deselected products by default
-  // eslint-disable-next-line jest/no-disabled-tests
-  it.skip('selects all products per default', async function () {
```

Comment:
we already have a similar test, 'applies defined default product selection,' so this test can be removed